server/queue/queue-rabbitmq/pom.xml

```diff
@@ -70,6 +70,15 @@
             <artifactId>testcontainers</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
I did this to code fast, but maybe we should rely on logback.

Reply:
of course